### PR TITLE
Add route for /org-user-view

### PR DESCRIPTION
**Pull request by boompttn**

### CODE REVIEW

File: src/modules/orgUserView/api.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+export const useOrganizationApi = (variables: { id: number }) =>
+  useQuery(
+    gql`
+      query getOrganization($id: Int!) {
+        organization(id: $id) {
+          id
+          name
+          isVerified
+          abbreviation
+          advisor
```
Comment (poom-kitti) [inline src/modules/orgUserView/api.ts]: Do u need this?

File: src/modules/orgUserView/api.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          associatedFaculty
```
Comment (poom-kitti) [inline src/modules/orgUserView/api.ts]: Do u need this?

File: src/modules/orgUserView/api.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+          description
+          facebookPage
+          instagram
+          lineOfficialAccount
+          email
+          contactFullName
+          contactEmail
+          contactPhoneNumber
+          contactLineId
```
Comment (poom-kitti) [inline src/modules/orgUserView/api.ts]: Do u need this?

File: src/modules/test/testData.ts [hunk 4]
```diff
@@ -328,6 +328,74 @@ export const eventsListData = [
...
+      location: "CU Sport Complex",
+      faculty: "ISE",
+      imgHash: "LEHV6nWB2yk8pyo0adR*.7kCMdnj",
+      img: "https://picsum.photos/201",
+      tags: ["Engineering", "Food", "Education"],
+      attendeeLimit: 999,
+      currentAttendee: 999
+    },
+    status: "Closed"
+  },
```
Comment (poom-kitti) [inline src/modules/test/testData.ts]: Instead of copy pasting same event, just create variable call "event" that have the event object then use that variable instead multiple times.

File: src/modules/orgUserView/OrgUserView.vue [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+    return { organization, events };
+  }
+});
+</script>
+
+<style scoped>
+.container {
+  max-width: 960px;
+}
```
Comment (poom-kitti) [inline src/modules/orgUserView/OrgUserView.vue]: There is already this class in index.css. Please refer there

File: src/modules/orgUserView/OrgUserView.vue [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    const organizationId = Number(route.params.id);
+    const { result, onError } = useOrganizationApi({
+      id: organizationId
+    });
+    onError(() => {
+      router.push("/404");
+    });
+
+    const organization = useResult(result, null, data => data.organization);
+    const events = useResult(result, null, data => data.organization.events);
```
Comment (poom-kitti) [inline src/modules/orgUserView/OrgUserView.vue]: Will this not just return id? Or are you waiting for new to do it?

File: src/modules/orgUserView/OrgUserView.vue [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+        Related events
+      </h1>
+      <div
+        class="card grid gap-4 justify-center md:justify-start // my-4 w-full"
+      >
+        <CardEvent
+          v-for="event in events"
+          :events="event"
+          :key="event.id"
+          class="flex"
```
Comment (poom-kitti) [inline src/modules/orgUserView/OrgUserView.vue]: Why is this necessary?

**Review by poom-kitti — DISMISSED** [review]
LGTM after fix and answer

File: src/modules/test/testData.ts (hunk 4 above)
Comment (boompttn) [reply src/modules/test/testData.ts]: Will revert to what it is since I've switched to using api

File: src/modules/orgUserView/OrgUserView.vue (hunk 7 above)
Comment (boompttn) [reply src/modules/orgUserView/OrgUserView.vue]: at first used flex, forgot to remove

File: src/modules/orgUserView/OrgUserView.vue (hunk 6 above)
Comment (boompttn) [reply src/modules/orgUserView/OrgUserView.vue]: change query to be the same as <Home /> where the Card is also used

File: src/modules/orgUserView/OrgUserView.vue [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+import "@/index.css";
+
+export default defineComponent({
+  name: "OrgUserView",
+  components: {
+    OrgCard,
+    CardEvent
+  },
+  setup() {
+    const route = useRoute();
```
Comment (new5558) [inline src/modules/orgUserView/OrgUserView.vue]: Can we move logic to customHooks?

**Review by new5558 — APPROVED** [review]
LGTM after fixed

File: src/modules/orgUserView/OrgUserView.vue (hunk 8 above)
Comment (boompttn) [reply src/modules/orgUserView/OrgUserView.vue]: Ok
